modules/brew/module.yml

```diff
@@ -0,0 +1,43 @@
+name: brew
+
+shortdesc: The brew module installs Homebrew (Brew) at build time and ensures the package manager remains updated and maintained.
+
+readme: https://raw.githubusercontent.com/blue-build/modules/main/modules/brew/README.md
+
+example: |
+
+  type: brew
+
+  # Installs Brew (Homebrew) to /home/linuxbrew/.linuxbrew via external download
+
+  install_brew: true # Optional - Default: true  - Expects type: boolean
+
+  # List of Brew packages to be installed
+
+  packages:
+
+    - ollama # Required - Default: n/a  - Expects type: list[string]
+
+  # Interval between Brew updates
+
+  update_interval: '6h' # Optional - Default: '6h' - Expects type: string
+
+  # Interval between Brew package upgrades
+
+  upgrade_interval: '8h' # Optional - Default: '8h' - Expects type: string
+
+  # Time delay after boot before first Brew update
+
+  wait_after_boot_update: '10min' # Optional - Default: '10min' - Expects type: string
+
+  # Time delay after boot before first Brew upgrade
+
+  wait_after_boot_upgrade: '30min' # Optional - Default: '30min' - Expects type: string
+
+  # If true, disables automatic activation of `brew-update.timer`.
+
+  disable_update: false # Optional - Default: false - Expects type: boolean
+
+  # If true, disables automatic activation of `brew-upgrade.timer`.
+
+  disable_upgrade: false # Optional - Default: false - Expects type: boolean
```

Comment:
We have a standard to use `kebab-case` for the properties of modules. Mind updating them to be that way?

Reply:
This is not solved as of now, so I'm resolving it.

Reply:
module.yml now uses kebab-case: https://github.com/blue-build/modules/pull/239/files#diff-e321f3fe6fca7490fb93078c575852784e86eccb2a746e9f33b29352444112f0

Reply:
Please update the README part that is highlighted in this review with kebab-cases

Reply:
I fixed it.